x-pack/plugins/ml/public/application/jobs/new_job/common/components/time_range_picker.tsx

```diff
@@ -25,6 +25,7 @@ interface Props {
 
 export const TimeRangePicker: FC<Props> = ({ setTimeRange, timeRange }) => {
   const mlContext = useMlContext();
+  // TODO should use fieldFormats instead
```

Comment:
This `TODO` is not for this PR. Just spotted it during the refactoring and left a note.